anndata/tests/test_dask.py

```diff
@@ -20,3 +55,69 @@ def test_dask_X_view():
     adata.X = da.ones((M, N))
     view = adata[:30]
     view.copy()
+
+
+def test_dask_write(adata, tmp_path, diskfmt):
+    import dask.array as da
+
+    pth = tmp_path / f"test_write.{diskfmt}"
+    write = lambda x, y: getattr(x, f"write_{diskfmt}")(y)
+    read = lambda x: getattr(ad, f"read_{diskfmt}")(x)
+
+    M, N = adata.X.shape
+    adata.obsm["a"] = da.ones((M, 10))
+    adata.varm["a"] = da.ones((N, 10))
```

Comment:
Could you make these randomly generated? This makes checking that the values are the same more strict.